### ISSUE TITLE
Comboboxes with VoiceOver

### ISSUE THREAD

**Issue by Veyfeyken** [accdc/tsg#9]
Hi, 

The comboboxes don't work with VoiceOver on MacOS Sierra and Safari 10. Using arrow down or up changes the selection in the list visually but nothing is read out. Tested on the 3 live demo's mentioned in your blogpost about role=”combobox”:

http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Native%20Inputs,%20Editable%20and%20Readonly)/demo.htm 

http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Native%20Inputs,%20Editable%20with%20Substring%20Match)/demo.htm

http://whatsock.com/tsg/Coding%20Arena/ARIA%20Comboboxes/ARIA%20Comboboxes%20(Simulated,%20Readonly)/demo.htm

**Comment by accdc** [comment]
Excellent, thanks for the heads up. I'll follow up with Apple to get bugs entered regarding this since the issue is a lack of aria-activedescendant support within the browser.

**Comment by accdc** [comment]
Hi,
This bug was confirmed as a lack of support within the Apple browser, the bug for which has been filed at
https://bugs.webkit.org/show_bug.cgi?id=167680
Which can be used to track the progress of the issue.

When support is correctly added by Apple this implementation will start working properly automatically without requiring any coding changes.

Thanks again for identifying the issue :)
